src/platform/android/CommissionableDataProviderImpl.cpp

```diff
@@ -34,6 +34,10 @@ using namespace chip::Crypto;
 
 namespace {
 
+#ifndef CHIP_DEVICE_CONFIG_USE_TEST_SPAKE2P_ITERATION_COUNT
+#define CHIP_DEVICE_CONFIG_USE_TEST_SPAKE2P_ITERATION_COUNT 1000
```

Comment:
Why did this need to be defined here?

Reply:
Ugh its annoying, its defined in `CHIPProjectConfig.h` only if  `CHIP_DEVICE_CONFIG_ENABLE_TEST_SETUP_PARAMS` is set to 1